workspace/TS100/Core/BSP/Pine64/Vendor/SoC/gd32vf103/Common/Source/Drivers/gd32vf103_spi.c

```diff
@@ -2,7 +2,7 @@
     \file  gd32vf103_spi.c
     \brief SPI driver
 
-    \version 2019-06-05, V1.0.0, firmware for GD32VF103
+    \version 2019-6-5, V1.0.0, firmware for GD32VF103

```

Comment:
This looks like some automated changes across all files.

Reply:
I have changed from using the drivers published on the gigadevice website to the ones from NMSIS

Moved to this one : https://github.com/Nuclei-Software/nuclei-sdk